src/backend/cuda/kernel/morph.hpp

```diff
@@ -59,17 +57,25 @@ inline __device__ void load2ShrdMem(T * shrd, const T * const in,
 
 // kernel assumes mask/filter is square and hence does the
 // necessary operations accordingly.
-template<typename T, bool isDilation, int windLen>
+// PLEASE MAKE SURE at least one of the following variables is not 0.
```

Comment:
A note would be fine. It would be better if you could document the template and function args.